src/group-configurations/group-configuration-container/index.jsx

```diff
@@ -0,0 +1,163 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+import { getConfig } from '@edx/frontend-platform';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow,
+  Hyperlink,
+  Icon,
+  IconButtonWithTooltip,
+} from '@edx/paragon';
+import {
+  DeleteOutline as DeleteOutlineIcon,
+  EditOutline as EditOutlineIcon,
+} from '@edx/paragon/icons';
+
+import ExperimentGroupStack from './ExperimentGroupStack';
+import TitleButton from './TitleButton';
+import UsageList from './UsageList';
+import messages from './messages';
+
+const GroupConfigurationContainer = ({ group, isExperiment, readOnly }) => {
```

Comment:
[question] It seems to me that there should be a component here similar to what was developed for the  course-outline page. I suggest correcting the button stroke (moving the identifier from the button), and perhaps adding a Tooltip component. What do you think?

<img width="843" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/c59cfe60-7f42-4cb0-85ba-46af9fac864f">